static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -217,11 +217,24 @@ aggregate_key
     }
 
 function_args
-  = arg1:key
-    args:(spaces comma spaces key)* {
+  = arg1:aggregate_param
+    args:(spaces comma spaces aggregate_param)* {
       return tc.tokenKeyAggregateArgs(arg1, args);
     }
 
+aggregate_param
+  = quoted_aggregate_param / raw_aggregate_param
+
+raw_aggregate_param
+  = param:[^()\t\n, \"]+ {
+    return tc.tokenKeySimple(param.join(''), false);
```

Comment:
In the frontend parser, we should probably just give these real token types instead of just masking them as `keySimple`

I would introduce a new TokenConverter method named `tokenKeyAggregateParam` which would look pretty similar to this one https://github.com/getsentry/sentry/blob/758ff2708ab354b070d79175706428579c860ae5/static/app/components/searchSyntax/parser.tsx#L387-L392 except that it would produce a token type of `Token.KeyAggregateParam`

these two types here would change to use the return from `tokenKeyAggregateParam` https://github.com/getsentry/sentry/blob/758ff2708ab354b070d79175706428579c860ae5/static/app/components/searchSyntax/parser.tsx#L419-L420